pipeline_tools/http_requests.py

```diff
@@ -16,6 +17,12 @@
 INDIVIDUAL_REQUEST_TIMEOUT = 'INDIVIDUAL_REQUEST_TIMEOUT'
 
 
+class RetryPolicy(retry_utils.Retry):
+    def __init__(self, retry_after_status_codes={301}, **kwargs):
+        super(RetryPolicy, self).__init__(**kwargs)
+        self.RETRY_AFTER_STATUS_CODES = frozenset(retry_after_status_codes | retry_utils.Retry.RETRY_AFTER_STATUS_CODES)
```

Comment:
It looks like `RETRY_AFTER_STATUS_CODES` includes 503 and some 4xx errors. We probably just want `self.RETRY_AFTER_STATUS_CODES = frozenset(retry_after_status_codes)` so we restrict to just retrying 301s here.

Reply:
Oh, I see! Thanks for catching that!